README.rst

```diff
@@ -1,130 +1,165 @@
-meta information in `setup.py <setup.py>`__ for current maintainers.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Project Status
---------------
+.. code-block:: bash
 
-|Linux Build Status| |Windows Build status|
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-|Scrutinizer Code Quality| |codecov.io|
+Store this file in the project's root directory and run coala:
 
-|Documentation Status| |Gitmate|
+.. code-block:: bash
 
-Newcomers Guide and Getting Involved
-------------------------------------
+    $ coala
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-We appreciate any help! Feel free to message us on
-`gitter <https://gitter.im/coala-analyzer/coala>`__. If you have any
-questions we're happy to help you!
+If you want to quickly test some settings without a ``.coafile``, you
+can run:
 
+.. code-block:: bash
 
-License
--------
+    $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True
 
-|AGPL|
+-----
+
+=======
+Support
+=======
+
+Feel free to contact us at our `Gitter channel <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
+
+You can also drop an email at our
+`mailing list <https://github.com/coala-analyzer/coala/wiki/Mailing-Lists>`__.
 
-This code falls under the GNU Affero General Public License as published
-by the Free Software Foundation, either version 3 of the License, or (at
-your option) any later version.
+|gitter|
 
-Please note that some files or content may be copied from other places.
-Most of them are GPL compatible. There is a small portion of code in the
-tests that falls under the Creative Commons license, see
-https://creativecommons.org/licenses/by-sa/3.0/deed.de for more
-information.
+-----
+
+================
+Getting Involved
+================
+
+If you would like to be a part of the coala community, you can check out our
+`Getting Involved <http://coala.readthedocs.io/en/latest/Getting_Involved/README.html>`__
+page or ask us at our active Gitter channel, where we have maintainers from
+all over the world. We appreciate any help!
+
+We also have a
+`newcomer guide <http://coala.readthedocs.io/en/latest/Getting_Involved/Newcomers.html>`__
+to help you get started by fixing an issue yourself! If you get stuck anywhere
+or need some help, feel free to drop a mail at our
+`newcomer mailing list <https://groups.google.com/d/forum/coala-newcomers>`__
+or contact us on Gitter.
+
+|gitter|
+
+-----
+
+=======
+Authors
+=======
+
+coala is maintained by a growing community. Please take a look at the
```

Comment:
this is super unhelpful. also that meta information is quite old, its only 3 guys in there and you say ``a growing community``. this is misleading :D again im asking as above, is there a method to list them here automatically, all the guys from coala, ordered by commits or something, like we have on the website, or it is too much :D ? or maybe link them to the page where you see the commits order? i just think that this information is vague :D